tests/integration/test_backup_restore_on_cluster/test.py

```diff
@@ -164,8 +163,15 @@ def test_replicated_database():
     node2.query("INSERT INTO mydb.tbl VALUES (2, 'count')")
     node1.query("INSERT INTO mydb.tbl VALUES (3, 'your')")
     node2.query("INSERT INTO mydb.tbl VALUES (4, 'chickens')")
+    node1.query("OPTIMIZE TABLE mydb.tbl ON CLUSTER 'cluster' FINAL")
```

Comment:
I guess it was trying to check backup of multiple parts, and now there's only one part

Reply:
Then we need to compare parts before and after backup, not data. I will add separate test on it.